docs/ref/contrib/gis/serializers.txt

```diff
@@ -41,6 +41,12 @@ The :ref:`fields <subset-of-fields>` option can be used to limit fields that
 will be present in the ``properties`` key, as it works with all other
 serializers.
 
+.. versionchanged:: 1.10
+
+    By default, the ``geojson`` serializer put the primary key of objects in
```

Comment:
Try to [make the note "self-contained"](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features) so we don't have to adjust the indentation when the `.. versionchanged:: 1.10` marker is removed.
chop "By default"
put->puts?
maybe the note would be better under the example, otherwise it might imply the example is related to this paragraph